lib/wibox/widget/slider.lua

```diff
@@ -90,6 +90,14 @@ local slider = {mt={}}
 -- @property bar_color
 -- @param color
 
+--- The bar (active) color.
+--
+--@DOC_wibox_widget_slider_bar_active_color_EXAMPLE@
```

Comment:
This should map to a file in `tests/examples/wibox/widget/slider/`. You can copy/paste one of the existing one, rename it then edit it to set the color. If you install `ldoc` then build awesome, you will be able to see the documentation with the new image for `active_color`. The images are generated automatically from those files.

Reply:
if I run 
```sh
 CMAKE_ARGS="$travis_build_cmd" make
```

and `make` successfully (with `ldoc` installed) then the build passes right?

Reply:
The Travis build does a lot more things than just compile Awesome. It also run the tests and do a lot of processing on a lot of files to generate a wide range of files (like the images used for the doc, a clean version of `rc.lua`, validated usage examples, the documentation, a module dependency graph and a bunch of other stuff).

But /generally/, if `make` passes and you open the path/to/build/dir/docs/index.html and it renders fine in your browser, you are most of the way there.

Reply:
I pushed the code after `luacheck`-ing and test building. The latest commit seems to have passed but the previous didn't, is that a problem?

```Checks failed for these commits:
5e327ce Use gears pattern instead of 2 shapes (hung, Thu Apr 18 21:47:00 2019 +0700)
93d271c Handle the handle_border_width when creating bg pattern (hung, Thu Apr 18 21:55:00 2019 +0700)
183d268 Draw slider's bar with different color depending on current value (hung, Thu Apr 18 11:12:21 2019 +0700)
efcc1bb Add beautiful variable in the comments (hung, Thu Apr 18 22:09:21 2019 +0700)